rwengine/src/script/modules/GTA3ModuleImpl.inl

```diff
@@ -890,23 +890,28 @@ void opcode_0051(const ScriptArguments& args) {
     thread->programCounter = thread->calls[--thread->stackDepth];
 }
 
+
 /**
     @brief  %5d% = create_player %1o% at %2d% %3d% %4d%
 
     opcode 0053
-    @arg model Model ID
-    @arg coord Coordinates
-    @arg player Player
+
+    Creates a player
+
+    @arg index Player index
+    @arg x X coordinate
+    @arg y Y coordinate
+    @arg z Z coordinate
+    @arg outPlayerHandle Created player
 */
-void opcode_0053(const ScriptArguments& args, const ScriptInt model, ScriptVec3 coord, ScriptPlayer& player) {
+void opcode_0053(const ScriptArguments& args, const ScriptInt index, ScriptVec3 coord, ScriptPlayer& player) {
     auto position = coord;
```

Comment:
Also getGround should be ``` if (p.z <= -100.0) ``` instead of
```
     if (p.z < -90.f) {
        p = args.getWorld()->getGroundAtPosition(p);
    }
```
to match original behavior